zone/spell_effects.cpp

```diff
@@ -7415,3 +7415,30 @@ void Client::BreakFeignDeathWhenCastOn(bool IsResisted)
 		MessageString(Chat::SpellFailure,FD_CAST_ON);
 	}
 }
+
+bool Mob::HarmonySpellLevelCheck(int32 spell_id, int target_id, Mob *target)
+{
+	//'this' = caster of spell
+	if (IsClient() && IsHarmonySpell(spell_id))
+	{
+		Mob *spell_target = nullptr;
+
+		if (target_id)
```

Comment:
Just a stylistic note for the future to just use brackets instead of implied if checks; it has caused issues in the past

```cpp
if (target_id) {
    // thing
}
```

vs

```cpp
if (target_id)
    // thing
```